src/compute/block.rs

```diff
@@ -348,13 +348,13 @@ fn perform_final_layout_on_in_flow_children(
 
     #[cfg_attr(not(feature = "content_size"), allow(unused_mut))]
     let mut inflow_content_size = Size::ZERO;
-    let mut committed_y_offset = resolved_content_box_inset.top;
+    let mut committed_offset = Point { x: resolved_content_box_inset.left, y: resolved_content_box_inset.top };
```

Comment:
This is a bit of a "nit" (and I would accept this PR without this change), but I don't think the name "committed_offset" really makes sense for the `x` axis here. Because it is supposed to capture the idea that we iterate over the items and add to ("commit to") the y offset each time. Which we don't do for the x axis.

Or to think about it another way. Note that `committed_offset.x` is mutable here even though it is never actually mutated.

My recommendation is that line 357 is changed to be `item.static_position = resolved_content_box_inset.left`. And then I believe all the other "committed offset"-related changes in this file can be undone (so bonus: much smaller diff!)